src/components/Login/Login.js

```diff
@@ -1,6 +1,38 @@
+import { useState, useEffect } from 'react';
 import PopupWithForm from '../PopupWithForm/PopupWithForm';
 
 function Login(props) {
+  const [errors, setErrors] = useState({ email: '', password: '' });
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  const validate = () => {
+    let errors = {};
```

Comment:
If a variable is declared using `let`, its value must change somewhere.
If a variable's value doesn’t change, it is declared using `const`.

All your variables should be declared using `const`